packages/time-picker/src/vaadin-time-picker.d.ts

```diff
@@ -112,6 +112,22 @@ export interface TimePickerEventMap extends HTMLElementEventMap, TimePickerCusto
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
  *
+ * ### Change events
```

Comment:
I'm experiencing some unexpected behavior:
1. adding an unparsable value to a time-picker with an initial value causes a "change"
2. Not sure why `unparsable => empty` causes an "unparsable-change" (even though it's documented). Isn't an empty value considered parsable?

https://github.com/vaadin/web-components/assets/1222264/677573b9-5735-4854-aa4d-7f02052a36af

Reply:
Ah, thanks for the explanation. It's fine as it is then.